src/test/java/org/opentest4j/AssertionFailedErrorTests.java

```diff
@@ -113,15 +133,33 @@ public void serializationWorksForAssertionFailedErrorWithMessageAndExpectedAndAc
+
+		AssertionFailedError error = deserializeAndCheck(Base64.decodeBase64(serializedVersion_1_0_0));
```

Comment:
I'm unclear if opentest4j's tests are compiled with Java 8 features or not, so if the following suggestion is impractical, please let me know!

Have you considered using Java 8's built-in [Base64](https://docs.oracle.com/javase/8/docs/api/java/util/Base64.html) instead of Apache Commons' one?

Reply:
Thanks for the suggestion! However, opentest4j is compatible with Java 6 at the moment.

Reply:
Okey dokey! :)